pkg/api/mlflow/controller/experiment.go

```diff
@@ -93,6 +93,10 @@ func (c Controller) DeleteExperiment(ctx *fiber.Ctx) error {
 		return err
 	}
 
+	if err := c.runService.DeleteRunsByExperiment(ctx.Context(), &req); err != nil {
```

Comment:
just an open question - do we need to use transaction here? @jgiannuzzi what do you think?

Reply:
I don't remember, does sqlite support this?

Reply:
Tx wrapper added